examples/SimpleRagas/SimpleRagas.ipynb

```diff
@@ -0,0 +1,615 @@
+    }
+   ],
+   "source": [
+    "from braintrust import EvalHooks\n",
+    "\n",
+    "\n",
+    "@braintrust.traced\n",
+    "async def generate_answer_e2e(question: str):\n",
+    "    retrieved_content = await fetch_top_k_relevant_sections(question)\n",
+    "    answer = await generate_answer_from_docs(question, retrieved_content)\n",
+    "    return dict(answer=answer, retrieved_docs=retrieved_content)\n",
+    "\n",
+    "\n",
+    "e2e_answer = await generate_answer_e2e(question)\n",
+    "print(e2e_answer.get(\"answer\"))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Perfect! Now that we have the whole system working, we can compute RAGAS metrics and try a couple improvements\n",
+    "\n",
+    "## Baseline ragas metrics with autoeval\n",
+    "\n",
+    "To get a large enough sample size for evaluations, we're going to use the synthetic test questions we generated in [ our earlier cookbook ](https://www.braintrust.dev/docs/cookbook/CodaHelpDesk). Feel free to check out that cookbook if you're interested in synthetic data generation, otherwise, you can just load them from this file location:\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 16,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "CODA_QA_PAIRS_LOC = \"https://gist.githubusercontent.com/nelsonauner/2ef4d38948b78a9ec2cff4aa265cff3f/raw/c47306b4469c68e8e495f4dc050f05aff9f997e1/qa_pairs_coda_data.jsonl\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "{'input': 'What is the purpose of starred docs in Coda?',\n",
+       " 'expected': 'Starring docs in Coda helps to mark documents of personal importance and organizes them in a specific section called My Shortcuts for easy access.',\n",
+       " 'metadata': {'document_id': '8179780',\n",
+       "  'section_id': 0,\n",
+       "  'question_idx': 0,\n",
+       "  'answer_idx': 0,\n",
+       "  'id': 0,\n",
+       "  'split': 'train'}}"
+      ]
+     },
+     "execution_count": 9,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "import json\n",
+    "\n",
+    "coda_qa_pairs = requests.get(CODA_QA_PAIRS_LOC)\n",
+    "\n",
+    "qa_pairs = [json.loads(line) for line in coda_qa_pairs.text.split(\"\\n\") if line]\n",
+    "\n",
+    "qa_pairs[0]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "RAGAS provides a variety of metrics - you can read an overview of each metric at the [Ragas metrics overview](https://docs.ragas.io/en/stable/concepts/metrics/index.html), but for the purposes of this guide, we'll show you how to calculate two scores we've found to be useful: Answer Correctness, which compares your system's answer to provided golden answer, and Context Recall, which compares the retrieved context to the information in the provided golden answer.\n",
```

Comment:
Capitalization is inconsistent. Should either be `RAGAS` everywhere, or `Ragas` everywhere. I assume it should never be fully lowercased (e.g. "Baseline ragas metrics...")